broker-daemon/relayer/identity.js

```diff
@@ -122,7 +122,12 @@ Identity.load = function (privKeyPath, pubKeyPath) {
  * @return {String} Pem-encoded public key without newlines or banners
  */
 function pubKeyToBase64 (fileContents) {
+  if (!fileContents) {
```

Comment:
It is possible to have docker generate blank certs, which would cause the line below to fail. It was a hard to debug so I added this error JIC